test/extended/prometheus/prometheus_builds.go

```diff
@@ -0,0 +1,191 @@
+package prometheus
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/prometheus/common/model"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/api/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature:Prometheus][builds] Prometheus", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLI("prometheus", exutil.KubeConfigPath())
+
+		execPodName, ns, host, bearerToken string
+		statsPort                          int
+	)
+
+	g.BeforeEach(func() {
+		ns, host, bearerToken, statsPort = bringUpPrometheusFromTemplate(oc)
+	})
+
+	g.Describe("when installed to the cluster", func() {
+		g.It("should start and expose a secured proxy and verify build metrics after a completed build", func() {
+			const (
+				appTemplate             = "https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/application-template.json"
```

Comment:
should use a template out of bindata like our other extended tests.

Reply:
update pushed - thx